tsconfig.base.json

```diff
@@ -9,15 +9,18 @@
     "strict": true,
     "target": "es5",
     "sourceMap": true,
+
     "plugins": [
       {
         "name": "typescript-tslint-plugin"
       }
     ],
+
     "paths": {
       "@datadog/browser-core": ["./packages/core/src"],
       "@datadog/browser-rum-core": ["./packages/rum-core/src"],
-      "@datadog/browser-rum": ["./packages/rum/src"]
+      "@datadog/browser-rum": ["./packages/rum/src"],
+      "@datadog/browser-rum-recorder": ["./packages/rum-recorder/src"]
```

Comment:
do we need recorder here?

Reply:
We don't, I just added it for completeness. Do you think I should remove it?

Reply:
Not necessarily, but for completeness we should also add logs

Reply:
On second thoughts, once rum-recorder will use rum-core, we would only need core-and rum-core here.
Maybe targeting that would be better, not sure if completeness add some value

Reply:
Agreed! I removed it.